src/background_process.c

```diff
@@ -164,6 +164,16 @@ void *background_process(void *ptr)
 		    log_to_disk(true);
 		    break;
 
+		case QTCRENTRY:
+
+		    store_recv_qtc(lan_message+2);
+		    break;
+
+		case QTCSENTRY:
+
+		    store_sent_qtc(lan_message+2);
+		    break;
+		    
```

Comment:
I don't see the context between your comment, and the modified code. The diff shows only the switch statement cases, but you wrote the "file misses". Should we need to clear it, or just simple regarded as solved?